lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -299,6 +307,24 @@
   stampTemplate('unoptimized-images-tmpl', document.body);
 }
 
+function deprecationsTest() {
+  stampTemplate('deprecations-tmpl', document.head);
+
+  // Some APIs atm are not triggering Chrome's deprecation messags. https://crbug.com/680832.
+  // try {
+  //   MediaStreamTrack.getSources(); // FAIL
+  // } catch(e) {}
+
+  // console.timeline(); // FAIL
+
+  // FAIL
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://lighthouse-viewer.appspot.com/data', false);
```

Comment:
can you do this to the local server? Ideally we wouldn't have any external connections for `npm run smoke`